infinex-multichain/staging/infinex-multichain-blast.toml

```diff
@@ -0,0 +1,99 @@
+options.CORE_INITIAL_PROXY_IMPLEMENTATION = "<%= imports.AccountFactory.imports.AccountFactory.contracts.InitialProxyImplementation.address %>"
+options.CORE_REVENUE_POOL = "<%= settings.REVENUE_POOL %>"
+options.CORE_USDC = "0x4300000000000000000000000000000000000003"
+options.CORE_CIRCLE_BRIDGE = "<%= settings.PLACEHOLDER_ADDRESS %>" # No CCTP on blast
+options.CORE_CIRCLE_MINTER = "<%= settings.PLACEHOLDER_ADDRESS %>" # No CCTP on blast
+options.CORE_DEFAULT_DESTINATION_CCTP_DOMAIN = "6"
+options.CORE_WORMHOLE_CIRCLE_BRIDGE = "<%= settings.PLACEHOLDER_ADDRESS %>" # No CCTP on blast
+options.CORE_DEFAULT_DESTINATION_WORMHOLE_CHAIN_ID = "30"                                                                                       # set the default domain to base for keeper bridging
+# ------- Solana Vars ------- #
+options.CORE_SOLANA_CCTP_DESTINATION_DOMAIN = "5"
+options.CORE_SOLANA_WALLET_SEED = "<%= hexlify('wallet_authority') %>"
+options.CORE_SOLANA_FIXED_PDA_SEED = "<%= hexlify('ProgramDerivedAddress') %>"
+options.CORE_SOLANA_WALLET_PROGRAM_ADDRESS = "0x0dd613b80b79792f2b77dbc1bc2a88a69da961324d890ed3aa9df8d517a500b3"           #gitleaks:allow
+options.CORE_SOLANA_TOKEN_MINT_ADDRESS = "0xc6fa7af3bedbad3a3d65f36aabc97431b1bbe4c2d2f6e0e47ca60203452f5d61"               #gitleaks:allow
+options.CORE_SOLANA_TOKEN_PROGRAM_ADDRESS = "0x06ddf6e1d765a193d9cbe146ceeb79ac1cb485ed5f5b37913a8cf5857eff00a9"            #gitleaks:allow
+options.CORE_SOLANA_ASSOCIATED_TOKEN_PROGRAM_ADDRESS = "0x8c97258f4e2489f1bb3d1029148e0d830b5a1399daff1084048e7bd8dbe9f859" #gitleaks:allow
+# ---------- Wormhole Chain Ids ---------- #
+options.CORE_WORMHOLE_CHAIN_ID_ARBITRUM = "23"
+options.CORE_WORMHOLE_CHAIN_ID_BASE = "30"
+options.CORE_WORMHOLE_CHAIN_ID_ETHEREUM = "2"
+options.CORE_WORMHOLE_CHAIN_ID_OPTIMISM = "24"
+options.CORE_WORMHOLE_CHAIN_ID_POLYGON = "5"
+options.CORE_WORMHOLE_CORE = "0xbebdb6C8ddC678FfA9f8748f85C815C556Dd8ac6"
+# ---------- Circle Constants ---------- #
+options.CORE_CCTP_DOMAIN_ARBITRUM = "3"
+options.CORE_CCTP_DOMAIN_BASE = "6"
+options.CORE_CCTP_DOMAIN_ETHEREUM = "0"
+options.CORE_CCTP_DOMAIN_OPTIMISM = "2"
+options.CORE_CCTP_DOMAIN_POLYGON = "7"
+
+# ---------- Setting Post deployment parameters  ---------- #
+[invoke.setInfinexProtocolConfigBeaconFee]
+target = ["infinex.InfinexProtocolConfigBeacon"]
+func = "setWithdrawalFeeUSDC"
+args = ["<%= settings.WITHDRAWAL_FEE_USDC %>"]
+from = "<%= settings.DEPLOYER %>"
+depends = ["clone.infinex"]
+
+[invoke.setTrustedRecoveryKeeper]
+target = ["infinex.InfinexProtocolConfigBeacon"]
+func = "setTrustedRecoveryKeeper"
+args = ["<%= settings.TRUSTED_RECOVERY_KEEPER %>", true]
+from = "<%= settings.DEPLOYER %>"
+depends = ["clone.infinex"]
+
+# [invoke.setFundsRecoveryActive]
+# target = ["infinex.InfinexProtocolConfigBeacon"]
+# func = "setFundsRecoveryActive"
+# args = []
+# from = "<%= settings.DEPLOYER %>"
+# depends = ["clone.infinex"]
+
+# [invoke.TransferOwnershipNewBeacon]
```

Comment:
I think this should have the transfer ownership steps left in, like what was done for prod. the other staging tomls had both transfer ownership and accept ownership, but that's because we're able to accept ownership for staging. but to most accurately mimic prod, we can't accept ownership so leave that out of the package.

Reply:
done